### DISCUSSION TITLE
What if CVE_SEARCH is enabled alone?

### DISCUSSION THREAD

**Discussion by noxing** [e-m-b-a/emba#619] [Q&A] [answered]
Is it possible to only enable CVE_SEARCH to scan the firmware? What should I do if I can? Thank you

**Comment by m-1-k-3** [comment, marked as answer]
cve_search is used from f20 module, but it uses multiple other modules as sources. You could try the following EMBA command:
```
sudo ./emba -f ~/firmware.bin -l ~/emba_logs -p ./scan-profiles/default-scan.emba -m s06 -m s08 -m s09 -m s24 -m s25 -m s115 -m s116
```

If you also want to get results from the kernel verification module you should add `-m s26`

Probably we should create a SBOM EMBA profile for such tasks.

**Comment by noxing** [comment]
If I do not run S115 and S116, what is the impact on the final vulnerability scan result? Can I add -m f20? I need this aggregated file

**Reply by m-1-k-3** [reply]
S115 and S116 are using emulation to identifiy version details - see https://github.com/e-m-b-a/emba/wiki/User-mode-Emulator. So, if you are interested in software components of the firmware you probably should run it. 
If you do not specify any f module all the f modules are enabled automatically ... You should be fine this way

**Reply by noxing** [reply]
> S115 and S116 are using emulation to identifiy version details - see https://github.com/e-m-b-a/emba/wiki/User-mode-Emulator. So, if you are interested in software components of the firmware you probably should run it. If you do not specify any f module all the f modules are enabled automatically ... You should be fine this way

Thank you very much. It works fine and meets my requirements

**Reply by m-1-k-3** [reply]
SBOM profile will be landed soon https://github.com/e-m-b-a/emba/pull/629
